### PR TITLE
`General`: Fix an issue when selecting a category tag in the exercises

### PR DESCRIPTION
**Pull request by edkaya**
#### General

- [ ] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [ ] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).

### Motivation and Context

Users encounter a bug when adding category tags to the exercises they create. The issue is as follows: An extra square appears in the category tags for exercises. This PR fixes the issue #8287.

### Description

One of the Angular Material "Chips" has global styling that adds an additional border around each category tag. This styling is overridden by removing its border property.

### Steps for Testing

1. Log in to Artemis
2. Navigate to Course Management
3. Navigate to Exercises
4. Create any exercise
5. Add a category
6. Verify that you don't have an extra square around the color tag (See the screenshots)

#### Code Review
- [x] Code Review 1
- [ ] Code Review 2

### Screenshots

- Before the fix: 
before 

- After the fix:
after

## Summary by CodeRabbit

- **Style**
	- Improved the visual consistency of the category selector by removing extra borders and stabilizing the 'x' button in the color picker interface.

### CODE REVIEW

**Review by egekurt123 — DISMISSED** [review]
Tested on ts2

**Review by JohannesWt — APPROVED** [review]
Code looks good

File: src/main/webapp/app/shared/category-selector/category-selector.component.scss [hunk 1]
```diff
@@ -34,7 +34,12 @@ jhi-category-selector {
     }
 }
 
-// Otherwise the x will jump when opening the colorpicker
+/* Otherwise each custom tag will have an additional border */
+.mat-mdc-standard-chip .mdc-evolution-chip__action--primary:before {
```
Comment (pzdr7) [inline src/main/webapp/app/shared/category-selector/category-selector.component.scss:38]: Can you double-check this line? Codacy is reporting an issue here:
image

Comment (edkaya) [reply src/main/webapp/app/shared/category-selector/category-selector.component.scss:38]: Hey, thanks a lot, you are right. I'll fix it right now.

**Review by pzdr7 — DISMISSED** [review]
Code

**Review by sarpsahinalp — APPROVED** [review]
Tested on ts1 works as expected!

**Review by florian-glombik — APPROVED** [review]
Tested on ts1, the border is no longer displayed and `!important` is needed at this place to overwrite the default `border-style: solid`

**Review by pzdr7 — APPROVED** [review]
Code

**Review by sarpsahinalp — APPROVED** [review]
Reapprove

**Review by JohannesWt — APPROVED** [review]
Code looks good 👍

**Review by egekurt123 — APPROVED** [review]
Reapprove

**Review by rabeatwork — APPROVED** [review]
Maintainer approved

**Pull request merged**